### ISSUE TITLE
Disable entering pause mode in 2d when holding the pen, since menus don't work <n/t>

### ISSUE THREAD

**Issue by gfodor** [mozilla/hubs#1550]

**Comment by misslivirose** [comment]
Hey @gfodor - can you confirm my understanding of this issue? 

When you're using the pen in 2D mode, the object menus don't work if you enter pause mode with space or tab. It does appear that freezing avatars and the avatar menu still works. 

If that's the correct understanding, I don't know that disabling the ability to enter pause mode is the right design for the fix - wouldn't this break the ability to freeze the room and manage a potentially harassing other party? I've definitely seen cases where people don't realize that they're still holding a pen, so I would suggest a fix where pausing force-drops the pen first, then gets you into pause mode.

If I'm misunderstanding the issue, please disregard.

**Comment by gfodor** [comment]
yeah, that's a fair point -- if it seems UX-wise that the menus could work while holding the pen then we should do that instead. it may depend on 2d vs 3dof vs 6dof too.
